strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
+        "mech": "workout-ready body",
+        "exosuit": "workout-ready body",
+        "phazon": "NEO",
+        "bolts": "high quality encryption",
+        "jester": "clown around town",
+        "salesman": "number 1 rated saleman1997",
+        "soul": "HeartShapedObject",
+        "souls": "hearts",
+        "bootstraps": "silly strings",
+        "mediocre": "$4.99",
+        "boring": "$4.99",
+        "cash": "wacky stacks",
+        "dump": "home-made storefront site",
+        "dumpster": "home-made storefront site",
+        "spending": "splurge",
+        "escape": "ride around town",
+        "fuck": "fifty percent off",
+        "inventory": "inventorium",
+        "garbage": "half-pr1ce sallamy",
+        "genocide": "total jackass stunts",
+        "massacre": "total jackass stunts",
+        "manipulation": "total jackass stunts",
```

Comment:
this is really uncommon terminology

Reply:
required for the reference, also, is it really uncommon? I guess i can add murder to the list though.